### ISSUE TITLE
Pillow 3.2.0 not installable on windows 10 / python 3.6

### ISSUE THREAD

**Issue by 77cc33** [shuup/shuup#1350]
Hello,

Your current shuup requirements not installable under Windows 10. Only from version 4.0 Pillow got python 3.6 support, maybe you can bump requirements version to something higher?

https://pillow.readthedocs.io/en/5.1.x/releasenotes/4.0.0.html

```
        ' using --disable-%s, aborting' % (f, f))
    ValueError: zlib is required unless explicitly disabled using --disable-zlib, aborting

    ----------------------------------------
Command "C:\Users\user\work\shuup\venv\Scripts\python.exe -u -c "import setuptools, tokenize;__file__='C:\\Users\\user\\AppData\\Local\\Temp\\pip-build-ml4qp47n\\pillow\\setup.py';f=getattr(tokenize, 'open', open)(__file__);code
=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record C:\Users\user\AppData\Local\Temp\pip-mwoni7g6-record\install-record.txt --single-version-externally-managed --compile --install-he
aders C:\Users\user\work\shuup\venv\include\site\python3.6\pillow" failed with error code 1 in C:\Users\user\AppData\Local\Temp\pip-build-ml4qp47n\pillow\
```

**Comment by chessbr** [comment]
Hi @77cc33 !
I guess we could try to do the same way easy_thumbnails does:
https://github.com/SmileyChris/easy-thumbnails/blob/master/setup.py#L55-L56
We didn't tested Pillow 4 so you could try and report to us the results you have :smiley:

**Comment by tulimaki** [comment]
I am pretty sure Shuup will work on newer Pillow too. We will look into that and update requirements if necessary. But with Windows you should be able to use something like this https://www.lfd.uci.edu/~gohlke/pythonlibs/#pillow to install Pillow. From Google you likel find some other options too. With Weasyprint there is some special steps with Windows too, but only required with pdf printouts.

**Comment by 77cc33** [comment]
Thanks for you replies guys!

From version 4.0 and up Pillow can be installed on windows directly from PIP, no need to download wheels, only VS Build Tools required.

I added PR  to fix your tests #1352

**Comment by tulimaki** [comment]
Solved.
